Rakefile

```diff
@@ -53,7 +53,7 @@ task :update_samples do
   require 'nokogiri'
   require 'yaml'
 
-  url = 'http://unicode.org/cldr/trac/browser/tags/release-26-d04/' +
+  url = 'http://unicode.org/cldr/trac/browser/tags/release-29/' +
```

Comment:
(According to Mark Davis who responded to my question on the cldr-users email list, the CLDR trac site was put behind auth because of a DoS attack)